src/azure-cli/azure/cli/command_modules/keyvault/_params.py

```diff
@@ -197,8 +197,10 @@ class CLIJsonWebKeyOperation(str, Enum):
             c.argument('not_before', default=None, help='Key not usable before the provided UTC datetime  (Y-m-d\'T\'H:M:S\'Z\').', type=datetime_type)
 
     with self.argument_context('keyvault key create') as c:
-        c.argument('kty', arg_type=get_enum_type(JsonWebKeyType), validator=validate_key_type)
-        c.argument('curve', arg_type=get_enum_type(JsonWebKeyCurveName))
+        c.argument('kty', arg_type=get_enum_type(JsonWebKeyType), validator=validate_key_type,
+                   help='The type of key to create. For valid values, see: https://docs.microsoft.com/en-us/rest/api/keyvault/createkey/createkey#jsonwebkeytype')
+        c.argument('curve', arg_type=get_enum_type(JsonWebKeyCurveName),
+                   help='Elliptic curve name. For valid values, see: https://docs.microsoft.com/en-us/rest/api/keyvault/createkey/createkey#jsonwebkeycurvename')
```

Comment:
Suggest to create aka.ms short links and use them in help message to keep it short.

Reply:
> Suggest to create aka.ms short links and use them in help message to keep it short.

@fengzhou-msft Good point. Actually this is a temporary solution for this case, in the future I hope the docs website can make these links automatically like wikipedia.